pkg/controllers/user/pipeline/controller/pipelineexecution/pipelineexecution.go

```diff
@@ -425,3 +516,49 @@ func (l *Lifecycle) reconcileRb(obj *v3.PipelineExecution) error {
 
 	return nil
 }
+
+func defaultNotificationMessage(execution *v3.PipelineExecution) string {
+	if execution == nil {
+		return ""
+	}
+	template := `
```

Comment:
do you think we can use go template for this? This way it would be easier to modify in the future